### DISCUSSION TITLE
Appropriate distribution for proportion data

### DISCUSSION THREAD

**Discussion by tommyclay** [bmcclintock/momentuHMM#73] [Q&A] [answered]
Dear Brett (and others),

I am trying to expand a step length and turning angle model to also include proportion data (values between 0 and 1) - that is the proportion of time recorded wet by a saltwater immersion logger (deployed alongside a GPS). I am aiming to refine models of seabird behaviour by including wet events, i.e. contact with the ocean surface. 

From looking at the vignette - https://cran.r-project.org/web/packages/momentuHMM/vignettes/momentuHMM.pdf - there doesn't appear to be a distribution that incorporates both 0 and 1 values and anything in between, unless I have missed something here? Unfortunately, the data are rather skewed towards 0 and 1 values, so using a normal distribution or something similar, is probably not possible.

I would appreciated anything that you could suggest. 

Many thanks,

Tommy

**Comment by bmcclintock** [comment, marked as answer]
Hi Tommy,

The beta distribution can be zero- and/or one-inflated, with additional zero-mass and one-mass parameters, respectively. See Section 2 and the "Harbour seals" example in the vignette. The harbour seal example used proportion of time diving, which was often zero (e.g. when a seal was hauled out or resting) but never one (because harbour seals need to breathe). Thus a zero-inflated beta distribution was used for that example. In your case, if particular behaviors are always associated with 0 or 1 values for "wet", then the zero- and one-mass parameters could be fixed accordingly (as was done for the zero-mass parameters in the harbour seal example).

Cheers,
Brett

**Comment by tommyclay** [comment]
Hi Brett,

Apologies for the long delay and thank you for your response. I indeed didn't notice that the Harbour Seals example was applicable to this case study so thanks for pointing that out. I'll have play around with the zero- and one-mass parameters. 

Many thanks,

Tommy
